tester_app/client/app/bundles/HelloWorld/startup/helloWorldAppServer.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+
+import createStore from '../store/helloWorldStore';
+import HelloWorld from '../containers/HelloWorld';
+
+const helloWorldAppServer = props => {
+  const store = createStore(props);
```

Comment:
see comments above, and mention 

// This is how the server redux gets hydrated with data.